safe-global/safe-wallet-web
# fix: Adjust overview widget on dashboard for new design

## What it solves

Updates the dashboard overview widget to support easy on-ramping

Figma: https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=8786%3A68251&mode=dev

## How this PR fixes it

## How to test it

1. Open a Safe
2. Go to the dashboard
3. Observe the updated design
4. Switch to a network where the Ramp Network app is activated
5. Observe a "Buy crypto" button

## Screenshots
<img width="649" alt="Screenshot 2023-11-09 at 13 57 10" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/d1d7f81f-be9c-4f2d-ab37-3d72c5ab6578">
<img width="624" alt="Screenshot 2023-11-09 at 13 57 27" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/269fbe23-38d2-4720-96eb-180a9d4edff3">

## Checklist
